frontend/src/features/resolve_errors/components/ResolveErrorsLayout.tsx

```diff
@@ -30,21 +30,42 @@ export function ResolveErrorsLayout() {
 
   const structure = getDataEntryStructure(dataEntry.data.model, election);
 
+  if (dataEntry.status == "first_entry_has_errors") {
+    return (
+      <>
+        <PageTitle title={`${t("resolve_errors.page_title")} - Abacus`} />
+        <header>
+          <section className="smaller-gap">
+            <PollingStationNumber>{pollingStation.number}</PollingStationNumber>
+            <h1>{pollingStation.name}</h1>
+            <Badge type={dataEntry.status} />
+          </section>
+        </header>
+        <Messages />
+        <main className={cls.resolveErrors}>
+          <StickyNav>
+            <DetailNavigation structure={structure} validationResults={dataEntry.validation_results} />
+          </StickyNav>
+          <article>
+            <Outlet />
+          </article>
+        </main>
+      </>
+    );
+  }
+
   return (
     <>
       <PageTitle title={`${t("resolve_errors.page_title")} - Abacus`} />
       <header>
         <section className="smaller-gap">
           <PollingStationNumber>{pollingStation.number}</PollingStationNumber>
           <h1>{pollingStation.name}</h1>
-          <Badge type="first_entry_has_errors" />
+          <Badge type={dataEntry.status} />
         </section>
       </header>
       <Messages />
       <main className={cls.resolveErrors}>
-        <StickyNav>
-          <ResolveErrorsNavigation structure={structure} validationResults={dataEntry.validation_results} />
-        </StickyNav>
```

Comment:
Without the `ResolveErrorsNavigation` I can't navigate to any other data entry section:

<img width="2554" height="1932" alt="Image" src="https://github.com/user-attachments/assets/96aa0f3a-de27-4d4d-933a-08b021e3442a" />